From 971390cac84afe4eeb7c39c8adca502e9da20436 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Wed, 11 Jan 2012 10:52:16 -0800
Subject: [PATCH 10/58] CDH-4069. Exclude clover jar from tarball/install.

The clover jar shouldn't be included in the tarball or packages, so
we're removing it here.
---
 build.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/build.xml b/src/build.xml
index 71e33eb..157ee14 100644
--- a/src/build.xml
+++ b/src/build.xml
@@ -516,7 +516,7 @@
 
     <!-- copy jar files -->
     <copy todir="${target.lib.dir}" preservelastmodified="true" flatten="true">
-      <fileset dir="${hive.root}" includes="*/*.jar, */*/*.jar" excludes="**/antlr-2*,**/antlr-3*,**/TestSerDe.jar"/>
+      <fileset dir="${hive.root}" includes="*/*.jar, */*/*.jar" excludes="**/antlr-2*,**/antlr-3*,**/TestSerDe.jar,**/clover-*.jar"/>
       <fileset file="${build.dir.hive}/cli/hive-cli-${version}.jar"/>
       <fileset file="${build.dir.hive}/common/hive-common-${version}.jar"/>
       <fileset file="${build.dir.hive}/ql/hive-exec-${version}.jar"/>
-- 
1.7.0.4

